package.json

```diff
@@ -213,10 +218,13 @@
     "unist-util-visit": "5.0.0",
     "use-local-storage-state": "19.5.0",
     "vite-plugin-solid": "2.11.0",
-    "vitest": "3.2.4",
+    "vitest": "4.0.12",
```

Comment:
Major upgrade. This should be upgraded in a separate PR?

Reply:
I think we'll eventually upgrade it with automatic renovate PRs.